packages/options/legacy/class.jetpack-options.php

```diff
@@ -93,7 +93,6 @@ public static function get_option_names( $type = 'compact' ) {
 			'identity_crisis_whitelist',    // (array)  An array of options, each having an array of the values whitelisted for it.
```

Comment:
Noting that I got heaps of phpcs errors from this file and had to use `--no-verify` to skip tests and commit:

```
FILE: ...mattic/jetpack/packages/options/legacy/class.jetpack-options.php
----------------------------------------------------------------------
FOUND 39 ERRORS AND 10 WARNINGS AFFECTING 41 LINES
----------------------------------------------------------------------
....
```